src/app.js

```diff
@@ -77,7 +77,7 @@ export default function (app) {
 
           } else {
             for (i = 0; i < hooks.onUpdate.length; i++) {
-              hooks.onUpdate[i](model, result, data)
+              hooks.onUpdate[i](model, result, data, actions)
```

Comment:
Maybe we should consider changing this to `model, data, actions, result`? That's similar to the action signature.